src/Command/BuildDocsCommand.php

```diff
@@ -129,7 +129,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         foreach ($this->finder as $file) {
             $htmlFile = str_replace([$this->sourceDir, '.rst'], [$this->htmlOutputDir, '.html'], $file->getRealPath());
-            if (!$this->filesystem->exists($htmlFile)) {
```

Comment:
It's not part of this command, but on general comment: I think this command should have some `--render-single-file=full.html` command. IF this is set, then, in addition to the normal rendering, we would also render a single `full.html` file (we would normally only use this with the `--parse-only` option while rendering one of the PDF's, but I don't think that should be a requirement).

Reply:
btw i don't like much `parse-only` name ;)

Reply:
Yea, me either... ``--parse-path`? "only" makes it sound like it's a boolean - everytime I see `string $parseOnly` I get confused :)